include/cantera/transport/GasTransport.h

```diff
@@ -115,8 +115,18 @@ class GasTransport : public Transport
      */
     virtual void getMixDiffCoeffsMass(doublereal* const d);
 
+    //! Return the polynomial fits to the viscosity of species i
+    virtual void getViscosityPolynomials(size_t i, double* coeffs) const;
+
+    //! Return the temperature fits of the heat conductivity of species i
+    virtual void getConductivityPolynomials(size_t i, double* coeffs) const;
+
+    //! Return the polynomial fits to the binary diffusivity of species pair (i, j)
+    virtual void getBinDiffusivityPolynomials(size_t i, size_t j, double* coeffs) const;
```

Comment:
It would be good to document the length of the `coeffs` vector, especially given that it is of variable size depending on the transport model variant. Also, links to the `fitProperties` and `fitDiffCoeffs` methods would be useful, since the forms of the fits are documented there.